package.json

```diff
@@ -42,12 +42,12 @@
     "@babel/eslint-parser": "7.18.9",
     "@babel/preset-env": "7.18.10",
     "@justeat/browserslist-config-fozzie": "2.0.0",
-    "caniuse-lite": "1.0.30001374",
     "@justeat/eslint-config-fozzie": "5.1.0",
     "@justeat/f-dom": "1.1.0",
     "@justeat/f-logger": "0.8.1",
     "@justeat/js-test-buddy": "0.4.1",
-    "@justeat/stylelint-config-fozzie": "2.3.0",
+    "@justeat/stylelint-config-fozzie": "file:.yalc/@justeat/stylelint-config-fozzie",
```

Comment:
You can't use yalc in PRs unfortunately. You can, however, run `npm pack` in the npm package you want to test (stylelint-config-fozzie in this case) to generate a `*.tgz` file, then copy that into the project you want to test it in and run `yarn add ./*.tgz -E` (the `*`'s here will be a generated file name) to reference the local test package and check it all in. CI should be able to read it all that way.

Reply:
Hey, this was just placeholder whilst I was waiting to publish the latest version of `stylelint-config-fozzie` :) Thats handy to know!

Reply:
(I thought Circle wouldn't build a draft)